### PR TITLE
Feature/listprofiles ascii

### LINKED ISSUE #15: Implement the full list-profiles subcommand
- [x] Connect CLI with Profile Handler
- [x] Create ASCII UI for listing the profiles

### PR DESCRIPTION
**Pull request by LeoLTM**
closes #15

### CODE REVIEW

File: src/main/java/de/dbtool/cli/subcommands/ListProfilesCommand.java [hunk 1]
```diff
@@ -13,6 +25,39 @@ public class ListProfilesCommand implements Runnable {
...
+
+            Profile[] profiles = profileHandler.listProfiles();
+            if(profiles == null || profiles.length == 0) {
+                System.out.println("No profiles found!");
+                return;
+            }
+            List<String[]> tableData = new ArrayList<>(profiles.length);
+            tableData.add(new String[] {"Name", "Hostname", "Port", "Database", "Username", "Password", "Type", "Driver"});
+            for(Profile p : profiles) {
+                tableData.add(new String[] {p.name, p.hostname, Integer.toString(p.port), p.dbName, p.username, p.password, p.type.toString(), p.driverPath});
```
Comment (mad201802) [inline src/main/java/de/dbtool/cli/subcommands/ListProfilesCommand.java]: Please print (*) instead of the password 😄

**Review by mad201802 — CHANGES_REQUESTED** [review]

File: src/main/java/de/dbtool/cli/subcommands/ListProfilesCommand.java (hunk 1 above)
Comment (LeoLTM) [reply src/main/java/de/dbtool/cli/subcommands/ListProfilesCommand.java]: Haha good point (= *) lol. 💯 
Thought about that but forgot it at the end ...

Comment (LeoLTM) [reply src/main/java/de/dbtool/cli/subcommands/ListProfilesCommand.java]: grafik

**Review by mad201802 — APPROVED** [review]
Passt

**Pull request merged**